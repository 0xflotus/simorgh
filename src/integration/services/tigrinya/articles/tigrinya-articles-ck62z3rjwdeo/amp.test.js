/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service tigrinya
 * @pathname /tigrinya/articles/ck62z3rjwdeo
 */

import runAmpTests from '../../../../pages/articles/ampTests';

describe('AMP tigrinya articles', runAmpTests);
