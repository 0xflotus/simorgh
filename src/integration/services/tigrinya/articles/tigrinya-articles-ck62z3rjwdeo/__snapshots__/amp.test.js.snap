// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP tigrinya articles A11y I can see the skip to content link 1`] = `"ናብቲ ትሕዝቶ ቀጽሉ"`;

exports[`AMP tigrinya articles AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP tigrinya articles AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP tigrinya articles Analytics ATI 1`] = `
Object {
  "requests": Object {
    "base": "https://logws1363.ati-host.net?",
    "pageview": "\${base}s=598343&s2=91&p=tigrinya.articles.ck62z3rjwdeo.page&r=\${screenWidth}x\${screenHeight}x\${screenColorDepth}&re=\${availableScreenWidth}x\${availableScreenHeight}&hl=\${timestamp}&lng=\${browserLanguage}&x1=[urn:bbc:optimo:ck62z3rjwdeo]&x2=[amp]&x3=[news-tigrinya]&x4=[ti]&x5=[\${sourceUrl}]&x6=[\${documentReferrer}]&x7=[article]&x8=[simorgh]&x9=[እትህቡና+ውልቃዊ+ሐበሬታ፡+ተሳትፎኹም+ኮነ+እተቕርቡዎ+ጥርዓንን+መልሰ+ዕንጋለን]&x11=[2019-11-27T11:25:04.923Z]&x12=[2019-11-27T11:25:04.923Z]&ref=\${documentReferrer}",
  },
  "transport": Object {
    "beacon": false,
    "image": true,
    "xhrpost": false,
  },
  "triggers": Object {
    "trackPageview": Object {
      "on": "visible",
      "request": "pageview",
    },
  },
}
`;

exports[`AMP tigrinya articles Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/tigrinya"`;

exports[`AMP tigrinya articles Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. ቢቢሲ፡ ንትሕዝቶ ካልኦት መርበባት ሓበሬታ ሓላፍነት ኣይወስድን። ብዛዕባ ምስ ናይ ደገ መርበባት እንገብሮ መላግቦታት ዘለና ኣረኣእያ ንምርዳእ ኣንብቡ።"`;

exports[`AMP tigrinya articles Header I can see the branding 1`] = `"BBC News, ትግርኛ"`;

exports[`AMP tigrinya articles I can see an image with a caption 1`] = `"https://ichef.bbci.co.uk/news/640/cpsdevpb/fa95/test/0985b6b0-1108-11ea-acaa-cd607a38aea4.jpg"`;

exports[`AMP tigrinya articles I can see an image with a caption 2`] = `"መግለጺ ስእሊ, ንመስርሕ ዕቃበን ኣጠቓቕማ ውልቃዊ ሐበሬታ ብዘማልእ ኣገባብ ይሕዞ"`;

exports[`AMP tigrinya articles SEO Canonical link 1`] = `"http://localhost:7080/tigrinya/articles/ck62z3rjwdeo"`;

exports[`AMP tigrinya articles SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP tigrinya articles SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP tigrinya articles SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP tigrinya articles SEO Heading level 1 1`] = `"እትህቡና ውልቃዊ ሐበሬታ፡ ተሳትፎኹም ኮነ እተቕርቡዎ ጥርዓንን መልሰ ዕንጋለን"`;

exports[`AMP tigrinya articles SEO Lang attribute 1`] = `"ti"`;

exports[`AMP tigrinya articles SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png",
      "width": 1024,
    },
    "name": "BBC News ትግርኛ",
    "noBylinesPolicy": "https://www.bbc.com/tigrinya/institutional-49283259#authorexpertise",
  },
  "dateModified": "2019-11-27T11:25:04.923Z",
  "datePublished": "2019-11-27T11:25:04.923Z",
  "headline": "እትህቡና ውልቃዊ ሐበሬታ፡ ተሳትፎኹም ኮነ እተቕርቡዎ ጥርዓንን መልሰ ዕንጋለን",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/tigrinya/articles/ck62z3rjwdeo",
    "@type": "WebPage",
    "name": "እትህቡና ውልቃዊ ሐበሬታ፡ ተሳትፎኹም ኮነ እተቕርቡዎ ጥርዓንን መልሰ ዕንጋለን",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png",
      "width": 1024,
    },
    "name": "BBC News ትግርኛ",
    "publishingPrinciples": "https://www.bbc.com/tigrinya/institutional-49283259",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png",
  "url": "http://localhost:7080/tigrinya/articles/ck62z3rjwdeo",
}
`;

exports[`AMP tigrinya articles SEO OG description 1`] = `"እትህቡና ውልቃዊ ሐበሬታ፡ ተሳትፎኹም ኮነ እተቕርቡዎ ጥርዓንን መልሰ ዕንጋለን"`;

exports[`AMP tigrinya articles SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png"`;

exports[`AMP tigrinya articles SEO OG image alt 1`] = `"BBC News ትግርኛ"`;

exports[`AMP tigrinya articles SEO OG locale 1`] = `"ti-ET"`;

exports[`AMP tigrinya articles SEO OG site name 1`] = `"BBC News ትግርኛ"`;

exports[`AMP tigrinya articles SEO OG title 1`] = `"እትህቡና ውልቃዊ ሐበሬታ፡ ተሳትፎኹም ኮነ እተቕርቡዎ ጥርዓንን መልሰ ዕንጋለን - BBC News ትግርኛ"`;

exports[`AMP tigrinya articles SEO OG type 1`] = `"article"`;

exports[`AMP tigrinya articles SEO OG url 1`] = `"http://localhost:7080/tigrinya/articles/ck62z3rjwdeo"`;

exports[`AMP tigrinya articles SEO Page title 1`] = `"እትህቡና ውልቃዊ ሐበሬታ፡ ተሳትፎኹም ኮነ እተቕርቡዎ ጥርዓንን መልሰ ዕንጋለን - BBC News ትግርኛ"`;

exports[`AMP tigrinya articles SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP tigrinya articles SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP tigrinya articles SEO Twitter creator 1`] = `"@bbcnews"`;

exports[`AMP tigrinya articles SEO Twitter description 1`] = `"እትህቡና ውልቃዊ ሐበሬታ፡ ተሳትፎኹም ኮነ እተቕርቡዎ ጥርዓንን መልሰ ዕንጋለን"`;

exports[`AMP tigrinya articles SEO Twitter image alt 1`] = `"BBC News ትግርኛ"`;

exports[`AMP tigrinya articles SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png"`;

exports[`AMP tigrinya articles SEO Twitter site 1`] = `"@bbcnews"`;

exports[`AMP tigrinya articles SEO Twitter title 1`] = `"እትህቡና ውልቃዊ ሐበሬታ፡ ተሳትፎኹም ኮነ እተቕርቡዎ ጥርዓንን መልሰ ዕንጋለን - BBC News ትግርኛ"`;
